docs/dev/Pagination-v2.md

```diff
@@ -285,3 +285,87 @@ OpenSearchExecutionEngine->>+ProjectOperator: getTotalHits
     ResourceMonitorPlan-->>-ProjectOperator: value
   ProjectOperator-->>-OpenSearchExecutionEngine: value
 ```
+
+### Plan Tree changes
```

Comment:
A couple recommended changes from above in the document: 
* Under #Scope, I think we can be more specific about what is out of scope vs what will be completed.  From what I understand: the document applies to all clauses (select...where...order by...) except for aggregation (group by).  Maybe some of those clauses aren't yet implemented (and that can be under the schedule) but the concepts still hold.  
* Fix: `The client receives an (error response](#error-response) if:` => `The client receives an [error response](#error-response) if:`
* Can we make the language consistent between initial page request and next page request:  `"cursor": /* cursor_id */` vs `"cursor": "<cursor_id>"`
* Maybe make section for `When Next Page Request is the Last Page` after `Next Page Request`?  And move the items that apply to the last page request to that section? 
* Please make the sub-categories under REST API and Design Diagrams consistent:  `Initial Query Request` vs `First Page` and `Next Page Request` vs `Second Page`.
* Under Design Diagrams/Second page: we include initial cursor verification in the sequence diagram.
* Do we need to update the `Total Hits` section for Peng's comments? 
*